osu.Game/Screens/OnlinePlay/Match/RoomSubScreen.cs

```diff
@@ -78,7 +78,7 @@ protected override void LoadComplete()
             managerUpdated = beatmapManager.ItemUpdated.GetBoundCopy();
             managerUpdated.BindValueChanged(beatmapUpdated);
 
-            UserMods.BindValueChanged(_ => updateMods());
+            UserMods.BindValueChanged(_ => UpdateMods());
```

Comment:
Note that due to this still being in the base implementation, it's called once in multiplayer as well (but will apply the last-known good state, rather than whatever `UserMods` just changed to).

This turns out to not be a noop for whatever equality or sequence comparison reason, but doesn't seem to have any adverse effects.

Reply:
Loose thought - if it is not a no-op in the mentioned case, won't it trigger `MultiplayerMatchSubScreen.onUserModsChanged()` again? Are we OK with that?